packages/openmrs/test/index.js

```diff
@@ -47,3 +59,239 @@ describe('execute', () => {
     });
   });
 });
+
+describe('request', () => {
+  it('should search for a patient', async () => {
```

Comment:
I think this test should be called `should GET with a query`. Because you're not really testing the search API, you're testing that the query to request gets appended to the url

Reply:
Otherwise this is a good test!

Reply:
Update the test